src/classes/Defaults/Trigger/Post/PostApproved.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Post upproved trigger
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\Trigger\Post;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+
+/**
+ * Post approved trigger class. Approved means published after review.
+ */
+class PostApproved extends PostTrigger {
+
+	/**
+	 * Constructor
+	 *
+	 * @param string $post_type optional, default: post.
+	 */
+	public function __construct( $post_type = 'post' ) {
+
+		parent::__construct( [
+			'post_type' => $post_type,
+			'slug'      => 'post/' . $post_type . '/approved',
+			// translators: singular post name.
+			'name'      => sprintf( __( '%s approved', 'notification' ), parent::get_post_type_name( $post_type ) ),
+		] );
+
+		$this->add_action( 'pending_to_publish', 10 );
+
+		// translators: 1. singular post name, 2. post type slug.
+		$this->set_description( sprintf( __( 'Fires when %1$s (%2$s) is approved', 'notification' ), parent::get_post_type_name( $post_type ), $post_type ) );
+
+	}
+
+	/**
+	 * Assigns action callback args to object
+	 *
+	 * @param object $post Post object.
+	 * @return mixed void or false if no notifications should be sent
+	 */
+	public function action( $post ) {
+
+		if ( $post->post_type !== $this->post_type ) {
+			return false;
+		}
+
+		$this->{ $this->post_type } = $post;
+
+		$this->author          = get_userdata( $this->{ $this->post_type }->post_author );
+		$this->last_editor     = get_userdata( get_post_meta( $this->{ $this->post_type }->ID, '_edit_last', true ) );
+		$this->publishing_user = get_userdata( get_current_user_id() );
```

Comment:
```suggestion
		$this->approving_user = get_userdata( get_current_user_id() );
```

Can we keep this property very specific and matching the group name?